Arm/Cosim.lean

```diff
@@ -99,7 +99,7 @@ def machine_to_regState (inst : BitVec 32) (str : String) : regState :=
   { inst, gpr, nzcv := flags[0]!, sfp }
 
 /-- Call the `armsimulate` script. -/
-def arm_cosim_test (input : regState) : IO String := do
+def arm_cosim_test (input : regState) (guid : Nat) : IO String := do
```

Comment:
What is `guid` being used for?

Reply:
I rename it to `fileBaseName` so we know that it's the filename that corresponds to the test.